awsmp/cli.py

```diff
@@ -56,10 +56,24 @@ def entity_list(entity_type, filter_visibility):
     """
     entity_list = _driver.list_entities(entity_type)
     t = prettytable.PrettyTable()
-    t.field_names = ["entity-id", "name", "visibility", "last-changed"]
+    if entity_type == "AmiProduct":
+        t.field_names = ["entity-id", "name", "visibility", "last-changed"]
+    else:
```

Comment:
we should be explicit on which entity types this is (e.g. in case we add SaaS later or other marketplace types)

Reply:
you have to give the entity type at the command line. so imo you can't get this wrong:

```
awsmp inspect entity-list AmiProduct|
```

vs.
```
awsmp inspect entity-list Offer
```

Reply:
what about direct library users? if we add more supported entities?